### PR TITLE
Update docs introduction

### PR DESCRIPTION
**Pull request by Gervwyk**
### What are the changes and their implications?
Update introduction for v4. 

## Checklist

- [x] Pull request is made to the "develop" branch
- [ ] Tests added
- [x] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

File: packages/docs/introduction.yaml [hunk 1]
```diff
@@ -25,60 +25,57 @@ _ref:
...
         properties:
           content: |
-            ##### TODO: This page needs to be updated.
+            Lowdefy makes is really easy to build web apps. With Lowdefy you can build CRUD apps, admin panels, internal tools, BI dashboards, workflow apps, websites, blogs, webforms or even documentation pages like this one.
 
-
-            Lowdefy simplifies creating software that helps manage internal data - think web apps, admin panels, BI dashboards, CRUD and workflow apps. Within most companies the need exists to connect to a variety of data sources and execute actions on such data. This must be done without compromising data security while respecting complex data integrity rules.
-
-            Lowdefy aims to enable companies to better control and monitor business data, by simplifying the process of creating internal tools. Lowdefy makes these tools easy to build, change and maintain, making it easier to evolve your internal tools with your business.
+            Lowdefy apps are defined in YAML config that follows a structured schema - this makes Lowdefy apps are easy to read, write and understand.
```
Comment (JohannMoller) [inline packages/docs/introduction.yaml:30]: ...this makes Lowdefy apps ~are~ easy to read, write and understand...

File: packages/docs/introduction.yaml [hunk 2]
```diff
@@ -25,60 +25,57 @@ _ref:
...
+            We've built Lowdefy to scale our own ability to develop and maintain web apps and internal tools for our customers.
 
-            Lowdefy is a open-source Apache-2 licensed project, free from vendor lock-in. Our business model is to build optional "nice-to-have" services best suited for Lowdefy applications, enriching the Lowdefy ecosystem.
+            Lowdefy is an open-source Apache-2 licensed project, free from vendor lock-in. We hope to see many community open-source Lowdefy apps and plugins in the future and actively nurture the Lowdefy community on [Github](https://github.com/lowdefy/lowdefy/discussions) and [Discord](https://discord.gg/QQY9eJ7A2D) to help other developers to get started with Lowdefy and grow the ecosystem.
 
-            Since Lowdefy apps are easy to manage in a git repository, we hope to see many community open-source Lowdefy apps in the future.
+            To date, more than 20 000 apps has been built using Lowdefy.
 
-            ### Deploy your Lowdefy app as you like
+            In the future plan to build optional "nice-to-have" services best suited for Lowdefy apps to further enriching the Lowdefy ecosystem.
```
Comment (JohannMoller) [inline packages/docs/introduction.yaml:70]: In the future **we** plan to build...

File: packages/docs/introduction.yaml [hunk 3]
```diff
@@ -25,60 +25,57 @@ _ref:
...
 
-            Host your Lowdefy app in your own environment as a docker image or even on Netlify using serverless functions. Serving a Lowdefy app, requires a the Lowdefy graphQl server and hosting the Lowdefy React app and blocks.
+            Currently we also provide Custom Software as a Service for larger customers, as well as support plans for those that need some extra help.
```
Comment (JohannMoller) [inline packages/docs/introduction.yaml:72]: ...support plans for those ~that~ **who** need some extra help.

File: packages/docs/introduction.yaml (hunk 2 above)
Comment (StephanieJKS) [reply packages/docs/introduction.yaml]: Change made in 2185f7c393e3cb32cf216ae86ec4d59f0b1a25a3

File: packages/docs/introduction.yaml (hunk 3 above)
Comment (StephanieJKS) [reply packages/docs/introduction.yaml]: Change made in 2185f7c393e3cb32cf216ae86ec4d59f0b1a25a3

File: packages/docs/introduction.yaml (hunk 1 above)
Comment (Gervwyk) [reply packages/docs/introduction.yaml]: Fixed

**Pull request merged**